packit.yaml

```diff
@@ -6,7 +6,7 @@ downstream_package_name: cockpit-machines
 actions:
   post-upstream-clone: make cockpit-machines.spec
   create-archive:
-    - test/download-dist --wait
+    - make dist-gzip DOWNLOAD_DIST_OPTIONS=--wait
```

Comment:
The env variable here does not need export? It's not clear to me yet, how the env variables are visible to the makefile target processes.

Reply:
This is passed as a "make variable", similar to `make V=1`. It has by and large the same effect as

    DOWNLOAD_DIST_OPTIONS=--wait make dist-gzip

or

    export DOWNLOAD_DIST_OPTIONS=--wait
    make dist-gzip

I'm happy to change the syntax if it looks weird to you, of course!

Reply:
It's fine of course. Just did not know the syntax.